src/components/tables/RowActions/Shared/UpdateEntity.tsx

```diff
@@ -112,17 +116,23 @@ function UpdateEntity({
         <SharedProgress
             name={entity.catalog_name}
             error={error}
-            renderError={() => (
-                <Alert
-                    icon={false}
-                    severity="error"
-                    sx={{
-                        maxHeight: 100,
-                        overflowY: 'auto',
-                    }}
-                >
-                    <DraftErrors draftId={draftId} />
-                </Alert>
+            logToken={logToken}
+            renderError={(errorProvided: any) => (
+                <>
+                    {errorProvided?.message ? (
+                        <Error error={errorProvided} hideTitle={true} />
+                    ) : null}
```

Comment:
If there is an error with a message then it is probably a Supabase error so go ahead and display that as well as any possible draft errors